src/slider/Slider.ts

```diff
@@ -109,10 +109,13 @@ export default class Slider extends SliderBase<SliderProperties> {
 			vertical = false,
 			verticalHeight = '200px'
 		} = this.properties;
-		const {
+		let {
 			value = min
 		} = this.properties;
 
+		value = value > max ? max : value;
+		value = value < min ? min : value;
```

Comment:
@smhigley just preference. Even if I used `Math.max` and `Math.min`, I'd want to check that the `value` was actually greater than the `max` or less than the `min` before calling them, and at that point, I might as well just use the `max` and `min` directly to avoid unnecessary indirection of a `Math` method.